token/cli/Cargo.toml

```
@@ -6,7 +6,7 @@ homepage = "https://spl.solana.com/token"
 license = "Apache-2.0"
 name = "spl-token-cli"
 repository = "https://github.com/solana-labs/solana-program-library"
-version = "2.4.0"
+version = "3.4.0"
```

Nit: no need to skip 3.0 - 3.3
```suggestion
version = "3.0.0"
```

---

Oops 😬